templates/cloud/openstack/index.html

```diff
@@ -637,7 +637,7 @@ <h3 class="p-heading--4">全自动的 OpenStack 部署和运营</h3>
         <h2>可扩展的容器层</h2>
         <h3 class="p-heading--4">在同一平台上运行容器和虚拟机</h3>
         <p>
-          通过在 OpenStack 上运行的 <a href="/containers/kubernetes">Kubernetes</a> 层来扩展您的云，并从统一的容器和传统工作负载管理方法中受益。
+          通过在 OpenStack 上运行的 <a href="https://ubuntu.com/kubernetes">Kubernetes</a> 层来扩展您的云，并从统一的容器和传统工作负载管理方法中受益。
```

Comment:
Can this not be `/kubernetes`?

Reply:
I think we should keep links internal where we can